p2p/pex/pex_reactor.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/pkg/errors"
 
-	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/go-amino"
```

Comment:
Should keep the old formatting. It's considered good form to use an explicit name if the package name differs from the URL name (`amino` vs `go-amino`), and to use a blank line between external and internal dependencies.

Reply:
for some reason my ide doesn't see the difference between amino and go-amino but I will make the change regardless